components/contact/ContactProvinceRow.js

```diff
@@ -1,5 +1,6 @@
 import Markdown from 'markdown-to-jsx'
-import { Collapse } from '@dts-stn/service-canada-design-system'
+// import { Collapse } from '@dts-stn/service-canada-design-system'
```

Comment:
All good! Just remove the comment before merging :)

Reply:
Right, will do.